test/acceptance/ignoreCase.robot

```diff
@@ -0,0 +1,19 @@
+*** Settings ***
```

Comment:
There already is a existing suite for this keyword, in ./test/acceptance/keywords/content_assertions.robot You have two options:
1) Move tests from here to the content_assertions suite
2) Move the `Element Should Contain` keyword test all in same and new suite. Also change the suite name to describe the whole keyword, not just the new feature.